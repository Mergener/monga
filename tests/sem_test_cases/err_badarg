type T = {
    f : float;
};

function foo(a : int, b : int): int {
    return a + b;
}

function main() {
    @foo(4, new T);
}