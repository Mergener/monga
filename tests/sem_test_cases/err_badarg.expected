[Error] Module sem_test_cases/err_badarg, line 10, column 19: Parameter type mismatch in 'foo' function call. Passed argument 2, of type 'T', cannot be assigned to a parameter of type 'int'.
Semantic errors were caught during analysis.
